daemon/entrypoint.sh

```diff
@@ -153,8 +153,13 @@ function start_osd {
             echo "No bootstrapped OSDs found; trying ceph-disk"
             osd_disk
          else
-            echo "Bootstrapped OSD(s) found; using OSD directory"
-            osd_directory
+            if [ -z "${OSD_DEVICE}" ]; then
+                echo "Bootstrapped OSD(s) found; using OSD directory"
+                osd_directory
+            else
+                echo "Bootstrapped OSD(s) found; using ${OSD_DEVICE}"
```

Comment:
hum if an OSD device is found why don't we try ceph-disk instead of osd_activate?